github/orgs_properties.go

```diff
@@ -23,8 +23,8 @@ type CustomProperty struct {
 	ValueType string `json:"value_type"`
 	// Whether the property is required.
 	Required *bool `json:"required,omitempty"`
-	// Default value of the property.
-	DefaultValue *string `json:"default_value,omitempty"`
+	// Default value of the property. Can be null, string or array of strings.
+	DefaultValue any `json:"default_value,omitempty"`
```

Comment:
@zyfy29 - for the a `null` to be passed through to the backend, we need to remove the `omitempty`.
Also, please make sure that there is a unit test where the value is `nil` (unassigned) so we demonstrate that the `null` is passed through to the backend.

```suggestion
	DefaultValue any `json:"default_value"`
```

Reply:
Done. Thanks!